plutus-benchmark/casing/bench/Bench.hs

```diff
@@ -19,7 +19,13 @@ import Data.Functor
 benchmarks :: EvaluationContext -> [Benchmark]
 benchmarks ctx =
     [ bgroup "casing"
-      [mkBMs "bool" Casing.casingBool
+      [ mkBMs "pairFstSnd" Casing.pairFstSnd
+      , mkBMs "pairCasing" Casing.pairCasing
+      , mkBMs "chooseUnit" Casing.chooseUnit
+      , mkBMs "unitCasing" Casing.unitCasing
+      , mkBMs "headList" Casing.headList
+      , mkBMs "headListCasing" Casing.headListCasing
+      , mkBMs "bool" Casing.casingBool
```

Comment:
These are not backported to `master`, right? So the whole benchmarking result is meaningless.

Reply:
No, the point is to compare builtin version of fst/snd pair to the casing version of it. Same with HeadList and ChooseUnit